azurerm/data_source_dns_zone.go

```diff
@@ -43,6 +43,29 @@ func dataSourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
+				Type:     schema.TypeString,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+			},
+
+			"registration_virtual_networks": {
+				Type:     schema.TypeList,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"resolution_virtual_networks": {
+				Type:     schema.TypeList,
+				Default:  nil,
+				Optional: true,
```

Comment:
since this is a Data Source and these values are returned (and not specified as inputs) - can we remove Default/Optional here?

Reply:
done.